index.html

```diff
@@ -4,14 +4,28 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/styles.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   <title>Aaron Keegan Developer Portfolio</title>
 </head>
 <body>
-
-  <!-- header
-  name:Aaron, sandwich icon -->
-  <!-- Section h1, h3, p (about) icons,
-  background image -->
-  
+  <header>
+    <nav class="top-nav">
```

Comment:
Make sure that you replicate Figma's design as much as possible 🧐 

### Your design 
![image](https://user-images.githubusercontent.com/15807343/153338744-213720e6-7eb9-4e72-aebf-8644e3c90441.png)

### Figma's design 
![Captura](https://user-images.githubusercontent.com/15807343/153338778-e9a9b7ec-8d2b-4674-b808-c18aa0ce3a70.PNG)

Please give your elements the appropriate margins so that your design resembles the best way to Figma's 😎

Also, kindly follow the fonts that the design has and implement them into your design 🔥